packages/core-ui/lib/statistics/Pie/index.tsx

```diff
@@ -0,0 +1,78 @@
+/* @license Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
+SPDX-License-Identifier: Apache-2.0 */
+import { useState } from "react";
+import "./index.scss";
+
+export type PieProps = {
+  title?: string;
+  titlePosition?: "top" | "right" | "bottom" | "left";
+  wrappedTitle?: boolean;
+  fixedTitle?: boolean;
+  size?: number;
+  perc?: number;
+};
+
+export const Pie = ({
+  title,
+  titlePosition = "top",
+  wrappedTitle = false,
+  fixedTitle = false,
+  size = 5,
+  perc = 42,
+}: PieProps) => {
+  const [showTitle, setShowTitle] = useState<boolean>(false);
+
+  return (
+    <>
+      <div
+        onMouseEnter={() => setShowTitle(true)}
+        onMouseLeave={() => setShowTitle(false)}
+        style={{
+          width: `${size}rem`,
+          height: title ? `${size + 2}rem` : `${size}rem`,
+        }}
+      >
+        {title && (
+          <div
+            className={`tooltip 
+              ${wrappedTitle ? "tooltip-wrap " : ""}
+              ${titlePosition + " "}
+              ${
+                fixedTitle ? "tooltip-show " : showTitle ? "tooltip-show " : ""
+              }`}
+            style={{
+              maxWidth: `${size * 2}rem`,
+            }}
+          >
+            {title}
+          </div>
+        )}
+        <svg
+          viewBox="0 0 100 100"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{
+            borderRadius: "50%",
+            border: "1px solid var(--network-color-primary)",
```

Comment:
Border can be removed as the staking dashboard no longer have borders, going more for the minimal approach.